src/field_theory/galois.lean

```diff
@@ -424,3 +424,14 @@ end
 end is_galois
 
 end galois_equivalent_definitions
+
+section is_alg_closure
+
+@[priority 100]
+instance is_alg_closure.is_galois (k K : Type*) [field k] [field K] [algebra k K]
```

Comment:
Does the fact that the proof is `{  }` mean that Lean can already find this instance? If `infer_instance` works there is no need to add it by hand.

Reply:
No, Lean cannot infer the instance by itself. The proof using `{  }` means that Lean can fill in by itself the fields for the instance: in this case, there are  instances for `is_alg_closure.normal` and `is_alg_closure.is_separable` so Lean can build the proof without any help needed.

I saw a proof like that at some point in `mathlib` and it took me a while to understand what was happening there ;)